front/src/components/Carousel.jsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from "react"
+import Image from "next/image"
+import { clsx } from "clsx"
+import config from "@/utils/config"
+
+const CarouselControllerIcon = ({ filled }) => {
+  if (filled) {
+    return (
+      <svg width="22" height="21" viewBox="0 0 22 21" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <path
+          d="M19.5 10.5C19.5 14.81 15.8053 18.5 11 18.5C6.19471 18.5 2.5 14.81 2.5 10.5C2.5 6.19005 6.19471 2.5 11 2.5C15.8053 2.5 19.5 6.19005 19.5 10.5Z"
+          fill="#A1D399"
+          stroke="#002203"
+          strokeWidth="5"
+        />
+      </svg>
+    )
+  }
+
+  return (
+    <svg width="22" height="21" viewBox="0 0 22 21" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path
+        d="M19.5 10.5C19.5 14.81 15.8053 18.5 11 18.5C6.19471 18.5 2.5 14.81 2.5 10.5C2.5 6.19005 6.19471 2.5 11 2.5C15.8053 2.5 19.5 6.19005 19.5 10.5Z"
+        stroke="#002203"
+        strokeWidth="5"
+      />
+    </svg>
```

Comment:
Same, can you use heroIcons ? Or at least create an component

Reply:
If we want to respect the mockup we need to use Figma's SVG. 
But created component for the icons so it looks like heroicons